src/content/core/okteto-manifest.mdx

```diff
@@ -168,3 +169,38 @@ You can learn more about the `destroy` section [here](reference/okteto-manifest.
 :::info
 Okteto recommends that you enable [Remote Execution](core/remote-execution.mdx) for your destroy commands
 :::
+
+## Test
+
+This section tells Okteto how to test your application.
+The Okteto CLI command `okteto test` will run tests using Okteto infrastructure via [Remote Execution](core/remote-execution.mdx#how-remote-execution-works).
+Tests can de defined independently using this section and will be executed sequentially. Tests can also depend on each other and be linked via a `depends_on` field which defines the order of test executions.
+
+Your `test` section might look like this:
+
+```yaml
+test:
+  setup:
+    image: okteto/golang:1
+    commands:
+      - "echo prepare tests"
+  unit:
+    depends_on:
+      - setup
+    image: okteto/golang:1
+    commands:
+      - "go test . -v"
+  integration:
+    depends_on:
+      - setup
+    context: integration
+    commands:
+      - "./run-tests.sh"
+```
+
+By default okteto runs all tests when you run `okteto test`. You also run a single section by passing it as an argument `okteto test unit`.
+
+
+:::note
+When running a dedicated test section, all `depends_on` fields will also be executed
```

Comment:
In this case, I think it's better to ignore dependencies. I visualize this for the inner loop, in case you want to re-run a particular test container, but the dependencies already passed

Reply:
I think it depends on the use case and how tests are structured. If I have, say, a `setup` stage that prepares the environment in some way for other stages dependencies are mandatory. I would find it strange if out of the box dependencies are not deployed. 
I do think it's a scenario we should support though. Maybe a `--no-deps` flag like `docker-compose`?

Reply:
Right, a `--no-deps` makes sense